tasks/main.yml

```diff
@@ -12,29 +12,29 @@
     - { path: "{{symfony2_project_root}}/shared/app/logs" }
     - { path: "{{symfony2_project_root}}/shared/web/uploads" }
 
-- name: Pull sources from the repository.
-  git: repo={{symfony2_project_repo}} dest={{symfony2_project_root}}/releases/{{symfony2_project_release}} version={{symfony2_project_branch}} accept_hostkey=yes
+# include does not allow templated path
+- include: "git_clone_export.yml"
+  when: symfony2_vcs_export_method == "git_clone"
 
-- name: Clean out versioning.
-  file: state=absent path={{symfony2_project_root}}/releases/{{symfony2_project_release}}/.git
-  when: symfony2_project_clean_versioning == true
+- include: "git_archive_export.yml"
+  when: symfony2_vcs_export_method == "git_archive"
 
 # will be replaced with symlink
 - name: Ensure logs directory is not present.
-  file: state=absent path={{symfony2_project_root}}/releases/{{symfony2_project_release}}/app/logs
+  file: state=absent path={{symfony2_project_root}}/releases/{{symfony2_project_release}}/{{symfony2_project_relative_root}}/app/logs
```

Comment:
Like in the other PR, there is an additional slash if the relative_path variable is not set.